models/src_amazon_ads.yml

```diff
@@ -39,11 +39,11 @@ sources:
             description: "{{ doc('state') }}"
 
       - name: ad_group_level_report
-        description: Campaign reports contain performance data broken down at the ad group and campaign level. 
+        description: Campaign reports contain performance data broken down at the ad group and campaign level.  
```

Comment:
The descriptions for these report type table read a bit strange. Can you adjust the wording of these descriptions to read more clearly.

Reply:
Same comment above for all the other `_report` table descriptions.

Reply:
Done.